### DISCUSSION TITLE
403 Error keeps popping up

### DISCUSSION THREAD

**Discussion by tenderassist** [pockethost/pockethost#253] [Q&A]
I made a web application using Next.js and pocketbase and have been using it for a while now with no issues, but for some reason today every time I try to do something on the web app it keeps returning a 403 error message. It never did this before and I haven’t changed any code in my web app or any settings in pocketbase. Any assistance would be greatly appreciated.

**Comment by benallfree** [comment]
Can you send along the exact network HTTP request and response please?

**Comment by benallfree** [comment]
Moving to #223
